compat/history-stream.js

```diff
@@ -21,28 +20,18 @@ exports.permissions = {
 
 exports.init = function (sbot, config) {
   sbot.createHistoryStream = Hookable(function createHistoryStream(opts) {
```

Comment:
I had to reimplement createHistoryStream using `toPullStream()` instead of `jitdb.paginate` because the former keeps track of `latestMsgPrecompaction` while we would have to manually do it with the latter.